website/src/config.ts

```diff
@@ -0,0 +1,162 @@
+const config = {
+  MIN_DESKTOP_WIDTH: 768,
+  WEB_MERCATOR_WKID: 3857,
+  MARKER_FILL_COLOR: [234, 202, 0, 0.5] as [number, number, number, number],
+  MARKER_OUTLINE_COLOR: [77, 42, 84, 1] as [number, number, number, number],
+};
+
+export default config;
+
+// old config
+// define([
+//     'app/App',
+
+//     'dojo/has',
+//     'dojo/request/xhr',
+//     'dojo/_base/Color',
+
+//     'esri/config',
```

Comment:
should all of this live in another file?

Reply:
This is just here as a reminder of what was in the legacy app. I'll clean it up once we start working.